component-classes/index.js

```diff
@@ -509,8 +509,8 @@ export const toaster = {
 
 // TOGGLE COMPONENTS:
 export const switchToggle = {
-  label: 'block relative h-24 w-44 cursor-pointer group',
-  labelDisabled: 'pointer-events-none',
+  base: 'block relative h-24 w-44 cursor-pointer group focusable rounded-full',
```

Comment:
`rounded-full` is only needed in React Switch (built with `button`+`span`), not in Vue Switch (built with `div`s), but I guess it's no harm to have an extra class here and there for now. In the future we'll probably build a Switch web component that will be reused in React and Vue packages, so this will get standarized anyway.